product_docs/docs/biganimal/release/administering_cluster/04_customizing_policy_ definitions.mdx

```diff
@@ -0,0 +1,221 @@
+
+In each of your BigAnimal-enabled Azure subscriptions, manually perform the following steps:
+
+!!! Note
+    You require *Microsoft.Authorizations/PolicyAssignments/write* permissions to be able to update policy initiatives (sets of policies) in Azure.
+
+
+1. In the Azure portal, enter **Policy** in the search box at the top and open the Policy service.
+
+1. Select **Compliance** on the left side of the Policy page. 
+
+1. On the Compliance page, set the Scope by selecting the ellipsis and then selecting all subscriptions. Click the **Select** button at the bottom of the Scope page to add your selection.
+
+1. You can see a list of all the policy initiatives (sets of policies) automatically assigned by Azure's onboarding process. The policy initiative for each subscription is labeled as *ASC Default (subscription: <Subscription_ID>)*
+
+1. Select a policy initiative in the list and click **Edit assignment**.  
+
+1. On the Edit Initiative Assignment page, click the **Parameters** tab.
+
+1. Clear the **Only show parameters that need input or review** check box.
+
+1. Configure your default ASC policy parameters to only allow BigAnimal's specific configurations. Use the parameter values specified in [Customizable policy definition parameters](#customizable-policy-definition-parameters) to update the parameters.
+
+1. Select the **Review + create** tab at the top of the wizard.
+
+1. Review your selections, then select **Create** at the bottom of the page.
+
+You're now ready to monitor, identify, and remediate non-compliant resources to improve the compliance state of the resources in your subscription.
+
+#### Customizable policy definition parameters
+
+While customizing Azure policies, refer to the following list of recommended parameters and values that are based on BigAnimal's resource configurations. 
+
+Use the values below each parameter while configuring the default ASC policy of a subscription. 
+
+!!! Note
+    
+    JSON values are provided wherever applicable.
+
+
+- **Allowed service ports list in Kubernetes cluster**:
+
+ Restrict services to listen only on ports used by BigAnimal, to secure access to the Kubernetes clusters. BigAnimal uses the following ports to send data requests to Kubernetes clusters:
+
+ ```js
+ ["5432",  "9402",  "443", "8080",
+   "9090",  "3000",  "8443",  "9443",  "9100", "9201", "8088"]
+ ```
+
+
+- **Allowed AppArmor profiles**:
+
+To limit container actions, BigAnimal recommends you to use the default AppArmor security profile.
+
+ ```js
+ ["runtime/default"]
+ ```
+ 
+
+- **Allowed capabilities**: 
+
+Restrict the capabilities to reduce the attack surface of containers in a Kubernetes cluster. 
+BigAnimal runs containers with very limited capability.
+
+ ```js
+ ["FOWNER"]
+ ```
+
+
+- **Max allowed memory bytes in Kubernetes cluster**: 
+
+Enforce the following container memory resource limits to prevent resource exhaustion attacks in a Kubernetes cluster:
+
+ `6Gi`
```

Comment:
This, and the CPU limit setting, seems like a bad idea. We could choose in the future to use resource limits to manage Postgres clusters, and this would completely break that.

Reply:
Right - we should remove both of these items.

Reply:
done